planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -145,7 +151,7 @@ class MoveGroup::MoveGroupImpl
                                                                                            false));
     waitForAction(place_action_client_, wait_for_server, move_group::PLACE_ACTION);
 
-    execute_service_ = node_handle_.serviceClient<moveit_msgs::ExecuteKnownTrajectory>(move_group::EXECUTE_SERVICE_NAME);
+    // execute_service_ = node_handle_.serviceClient<moveit_msgs::ExecuteKnownTrajectory>(move_group::EXECUTE_SERVICE_NAME);
```

Comment:
The problem is the default capabilities are given by rosparam at here.
 https://github.com/ros-planning/moveit_ros/blob/kinetic-devel/move_group/src/move_group.cpp#L107

I was not sure how the execute service could be replaced by the action.
But in @rhaschke 's [comment](https://github.com/ros-planning/moveit_ros/pull/719#issuecomment-233404206), he suggests the process of migration and I will follow that.